packages/ra-core/src/dataProvider/withDataProvider.tsx

```diff
@@ -1,64 +0,0 @@
-import * as React from 'react';
```

Comment:
oops, this one was documented, wo we need to add aline in the upgrade guide about it.

Reply:
And remove it from `Reference.md`